.github/workflows/tests.yml

```diff
@@ -33,3 +41,17 @@ jobs:
         if: matrix.python-version == '3.8'
       - name: Build wheel
         run: python3 -m build --sdist --wheel .
+      - name: Run Jupyter
+        run: |
+          jupyter nbconvert --to html --TagRemovePreprocessor.remove_cell_tags pip --output TlkAgg-Categorical.html --execute examples/TlkAgg-Categorical.ipynb
+          jupyter nbconvert --to html --TagRemovePreprocessor.remove_cell_tags pip --output Readability-Pairwise.html --execute examples/Readability-Pairwise.ipynb
```

Comment:
Is it possible to convert all examples in `/examples` folder in one command? Imagine we have more and more examples, each time we need to add them with the same pattern

Reply:
Good idea; fixed.